docker/Dockerfile

```
@@ -17,7 +17,7 @@ RUN /bin/bash ./install.sh
 
 COPY . /plotman
 
-RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman && deactivate"] 
+RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman && pip install supervisor && deactivate"]
```

We are already installing into an existing env which pip isn't the best at.  But we can still let pip know all the additional things it needs to install at once.

```suggestion
RUN ["/bin/bash", "-c", "source ./activate && pip install /plotman supervisor && deactivate"]
```